src/icons/EarnAave.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import Svg, { ClipPath, Defs, G, LinearGradient, Path, Rect, Stop } from 'react-native-svg'
+
+const EarnAave = () => (
+  <Svg width={34} height={34} viewBox="0 0 34 34" fill="none">
+    <Rect width={32} height={32} rx={16} fill="#F1FDF1" />
```

Comment:
It looks like this is also `Colors.successLight` should we pass this color instead of the hardcoded value?